app/views/lists/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Lists</h1>
+
+
+<% @lists.each do |list| %>
+<div>
+  <h2><%= list.title %></h2>
+  <p> <%= list.description %> </p>
+  <% list.books.each_with_index do |book, index| %>
```

Comment:
@samnang do you have any thoughts on the correct way to do this? We were trying to limit the shown `books` association to 6-10 items per `list` and couldn't find an easy solution with ActiveRecord.